sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -588,6 +600,18 @@ func resourceSumologicMonitorsLibraryMonitorRead(d *schema.ResourceData, meta in
 		return nil
 	}
 
+	fgpResponse, fgpErr := c.GetCmfFgp(fgpTargetType, monitor.ID)
+	if fgpErr != nil {
+		// if FGP endpoint is not enabled (not implemented), we should suppress this error
+		if len(HasErrorCode(fgpErr.Error(), []string{"not_implemented_yet"})) == 0 {
```

Comment:
The list of error codes should have `"api_not_enabled"` too?

Reply:
done